### DISCUSSION TITLE
Multistream

### DISCUSSION THREAD

**Discussion by DenisStadnichenko** [shogo4405/HaishinKit.kt#190] [Q&A]
Hi, can you please advise me if it is possible to create a multistream on android?

`  cameraView.attachStream(twitchStream)
   cameraView.attachStream(youTubeStream)`

Is there any possibility to attach 2 streams to one view?

**Comment by shogo4405** [comment]
No.

**Comment by DenisStadnichenko** [comment]
Got it, thanks for the answer
